.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+ignore =
+    # E126 - continuation line over-indented for hanging indent
+    E126,
+    # N813 - camelcase imported as lowercase
```

Comment:
N813 is allowed because of this frequent line: `from h.i18n import TranslationString as _`